core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -509,6 +507,10 @@ private class LiveRDD(val info: RDDInfo) extends LiveEntity {
 
   private val distributions = new HashMap[String, LiveRDDDistribution]()
 
+  def isEmpty(): Boolean = {
+    memoryUsed == 0L && diskUsed == 0L && partitions.isEmpty && distributions.isEmpty
```

Comment:
Shouldn't this be the equivalent of `storageLevel == None` instead? That's what 2.2 does:

https://github.com/apache/spark/blob/branch-2.2/core/src/main/scala/org/apache/spark/storage/StorageStatusListener.scala#L52